packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -122,6 +122,61 @@ describe('Auth', () => {
         // @ts-expect-error private method
         expect(() => auth.isTokenExpired()).toThrow()
     })
+    test('hasSFRAAuthStateChanged', () => {
+        // Should return true if refresh_token keys are different
+        const auth = new Auth(config)
+        // @ts-expect-error private method
```

Comment:
Is there a way to call these functions indirectly using `public` methods to avoid hitting `private method` error a million times here?

Reply:
None of these methods need to be accessed outside `Auth` module and are all private. We would  need to remove testing the functions independently and directly test `ready()` as a whole to avoid private method errors.